### PR TITLE
Feature/sched e transition

### PR DESCRIPTION
**Pull request by jontours**
This pull request was created from a previous commit on my other open pull request: #1694.  This pull request has changes for all warehouse tables I worked on.  Opened this pull request so we can work on sched_e changes as the data becomes available.

### CODE REVIEW

**Comment by LindsayYoung** [comment]
Can we put [DON'T MERGE] in the front of the title for now?

**Comment by jontours** [comment]
Sure!

**Comment by LindsayYoung** [comment]
We need `NML_SCHED_E - 24/28`

**Comment by LindsayYoung** [comment]
Documenting verbal conversation:
Things look pretty good, Jon is going to merge the two endpoints back to one and double check that fixes the sched E problems we have been seeing.

Approving this is going to be my first priority and I will double check for sql optimizations and verify the filters and sorts are working.

We hope to have this as a mini-release early next week.

File: webservices/docs.py [hunk 1]
```diff
@@ -434,8 +434,8 @@
 a political party or its agents.
 
 Aggregates by candidate do not include 24 and 48 hour reports. This ensures we don't double count expenditures
-and the totals are more accurate. You can still find the information from 24 and 48 hour reports in
-`/schedule/schedule_e/`.
+and the totals are more accurate. You can find the information from 24 and 48 hour reports in
+`/schedule/schedule_e/notice`.
```
Comment (LindsayYoung) [inline webservices/docs.py]: We changed this, right?

**Comment by LindsayYoung** [comment]
Here is the one that was not returning results for me:
http://localhost:5000/v1/schedules/schedule_e/?per_page=20&sort=-expenditure_date&is_notice=true

**Comment by jontours** [comment]
I was not able to reproduce that, trying updating the local dev data base (`prepare_schedule_e.sql`) and try testing again.  Let me know if that doesn't work.

File: webservices/docs.py (hunk 1 above)
Comment (jontours) [inline webservices/docs.py]: yup, I'll go ahead and change it back.

**Pull request merged**